test/download-test.js

```diff
@@ -198,6 +198,7 @@ test('error during download should fail with no dangling temp file', function (t
 test('should fail if abi is system abi with invalid binary', function (t) {
   var opts = getOpts()
   opts.abi = process.versions.modules
+  opts.runtime = 'node'
```

Comment:
Maybe `getOpts` should set the default runtime?

Reply:
Sure, maybe a bit cleaner.